### ISSUE TITLE
Highlight a piece of text as a tag

### ISSUE THREAD

**Issue by ElizaLo** [mytcat/Form-to-google-spreadsheet#36]
<img width="716" alt="Снимок экрана 2020-01-14 в 11 50 08" src="https://user-images.githubusercontent.com/36056528/72333341-4bb71100-36c4-11ea-9694-d9143b9e30d0.png">

 Finds a **subword** in the word, which is **not** correct. 
Sometimes it's not even a subword, it's just a **part of the word** (_for example_, in some articles we can see highlight **" 're"** as a tag, etc.)

**Comment by mytcat** [comment]
Time Estimation: 3hours

**Comment by AlexUdodenko** [comment]
а это баг или фича?

**Comment by ElizaLo** [comment]
> а это баг или фича?

Я и @zhuker считаем, что это баг.

**Comment by mytcat** [comment]
Да, это баг в js либе, которая занимается подсветкой. Надо переписывать либу или писать что-то своё. Если получится сделать быстрее, я сделаю "re-estimate".

**Comment by AlexUdodenko** [comment]
Approved
